examples/multioutput/plot_classifier_chain_yeast.py

```diff
@@ -0,0 +1,110 @@
+"""
+============================
+Classifier Chain
+============================
+Example of using classifier chain on a multilabel dataset.
+
```

Comment:
I guess my preference is to keep the text concise and focused on the example. If there is a specific point you think we should make here I'd consider adding a sentence or two. Otherwise I'm inclined to leave it as is.

I think we should stay away from dealing with regression models in this PR. But I agree that is a good direction to go.